ee/modules/110-istio/hooks/discovery_versions_to_install_test.go

```diff
@@ -15,19 +15,45 @@ import (
 	"sigs.k8s.io/yaml"
 
 	"github.com/deckhouse/deckhouse/go_lib/dependency"
+	"github.com/deckhouse/deckhouse/go_lib/telemetry"
 	. "github.com/deckhouse/deckhouse/testing/hooks"
 )
 
 var _ = Describe("Istio hooks :: discovery_versions_to_install ::", func() {
 	f := HookExecutionConfigInit(`{"istio":{}}`, "")
 
+	assertNoMetrics := func(f *HookExecutionConfig) {
+		m := f.MetricsCollector.CollectedMetrics()
+		Expect(m).To(HaveLen(0))
+	}
+
+	assertTelemetryMetrics := func(f *HookExecutionConfig, versions []string) {
```

Comment:
You pass single version to `assertTelemetryMetrics` in all places it is used. Can we remove the need for the slice?

Reply:
yes

Reply:
done